buildbot/configure.py

```diff
@@ -182,6 +185,7 @@ def main():
     parser.add_argument("--libcxx-include", metavar="LIBCXX_INCLUDE_PATH", help="libcxx include path")
     parser.add_argument("--libcxx-library", metavar="LIBCXX_LIBRARY_PATH", help="libcxx library path")
     parser.add_argument("--use-lld", action="store_true", help="Use LLD linker for build")
+    parser.add_argument("--llvm-external-projects", help="Add external projects to build. Add as comma seperated list.")
```

Comment:
Why "comma separated"?
"semicolon separated" should simplify the process. Right?

Reply:
Comma separated is used because bash interprets `;` as a command separator. So the full list is not sent to configure.py and bash tries to use the projects name as a command.

Reply:
I thought we can use surround semicolon with quotes:
```bash
./configure --llvm-external-projects="projectA;proejctB"
```

Anyway, I'm okay with comma separator as well. Thanks for clarification.